cli/deploycheck/policy.go

```diff
@@ -60,6 +60,10 @@ func PolicyCompatible(org string, userPw string, nodeId string, nodeArch string,
 		policyCheckInput.NodePolicy = &np
 	}
 
+	if nodeType == "" {
```

Comment:
nodeId may also be empty. So please use: 
if nodeType == "" && policyCheckInput.NodeId != ""

Reply:
Added nodeId check